README.md

```diff
@@ -257,6 +253,6 @@ I would like to acknowledge:
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./License) licensed.
+This project is [MIT](https://github.com/muneeb3123/Fast-Lane-Racers-back-end/blob/dev/LICENSE.md) licensed.
```

Comment:
- [ ] The link is still broken because the file name is wrong.  

![image](https://github.com/muneeb3123/Fast-Lane-Racers-back-end/assets/57421823/999c994d-911e-4db8-85f6-0eccac6dfebc)


Kindly check the capitalization and include the correct exact file name so the link works.  Don't forget to test it. 👍